samples/nextjs/scripts/generate-component-factory.ts

```diff
@@ -79,15 +81,15 @@ function writeComponentFactory() {
 }
 
 function getComponentList(path: string): (PackageDefinition | ComponentFile)[] {
-  const components = getItems<PackageDefinition | ComponentFile>(
+  const components = getItems<PackageDefinition | ComponentFile>({
     path,
-    (path, name) => ({
+    resolveItem: (path, name) => ({
       path: `${path}/${name}`,
       componentName: name,
       moduleName: name.replace(/[^\w]+/g, ''),
     }),
-    (name) => console.debug(`Registering JSS component ${name}`)
-  );
+    cb: (name) => console.debug(`Registering JSS component ${name}`),
```

Comment:
Is "cb" common for callback naming convention? Or should we go with full "callback"?

Reply:
yes, cb - means callback

Reply:
https://stackoverflow.com/questions/31780872/what-is-cb-in-node

Reply:
https://derickbailey.com/2016/02/15/what-do-you-name-the-callback-function/